packages/@react-aria/selection/src/useSelectableCollection.ts

```diff
@@ -201,32 +204,41 @@ export function useSelectableCollection(options: SelectableCollectionOptions): S
         }
         break;
       case 'PageDown':
-        if (delegate.getKeyPageBelow) {
+        if (delegate.getKeyPageBelow && !isKeyboardNavigationDisabled) {
           e.preventDefault();
           let nextKey = delegate.getKeyPageBelow(manager.focusedKey);
           navigateToKey(nextKey);
         }
         break;
       case 'PageUp':
-        if (delegate.getKeyPageAbove) {
+        if (delegate.getKeyPageAbove && !isKeyboardNavigationDisabled) {
           e.preventDefault();
           let nextKey = delegate.getKeyPageAbove(manager.focusedKey);
           navigateToKey(nextKey);
         }
         break;
       case 'a':
-        if (isCtrlKeyPressed(e) && manager.selectionMode === 'multiple' && disallowSelectAll !== true) {
+        // disabled navigation also means disabling selection i think, otherwise trying to type 'a' into a textfield would be disastrous
+        if (isCtrlKeyPressed(e) && manager.selectionMode === 'multiple' && disallowSelectAll !== true && !isKeyboardNavigationDisabled) {
           e.preventDefault();
           manager.selectAll();
         }
         break;
       case 'Escape':
-        e.preventDefault();
-        if (!disallowEmptySelection) {
-          manager.clearSelection();
+        // disabled navigation also means disabling selection i think, similar reason to case 'a' but trying to exit out of a resizer
+        if (!isKeyboardNavigationDisabled) {
+          e.preventDefault();
+          if (!disallowEmptySelection) {
+            manager.clearSelection();
+          }
         }
         break;
       case 'Tab': {
+        if (isKeyboardNavigationDisabled && delegate.getKeyRightOf && delegate.getKeyLeftOf) {
+          e.preventDefault();
+          let nextKey = delegate.getKeyRightOf(manager.focusedKey);
```

Comment:
I don't feel great about this being the right solution for Tab/Shift+Tab. Given previous comment in useGrid. However, we'll want to have the same behavior across grid/collections when hitting Tab in edit mode. I think it should put you back on the containing cell (or child of the cell depending on the strategy). However, this has no concept of actual dom, only focusedKeys, and actual focusing happens in useSelectableItem. I could watch in useSelectableItem for changes to isKeyboardNavigationDisabled and re-focus the cell when it becomes false. This would require sending either the state in to useSelectableItem or some other option